src/event.js

```diff
@@ -677,30 +683,24 @@ jQuery.Event.prototype = {
 	isImmediatePropagationStopped: returnFalse,
 
 	preventDefault: function() {
-		var e = this.originalEvent;
-
 		this.isDefaultPrevented = returnTrue;
 
-		if ( e && e.preventDefault ) {
-			e.preventDefault();
+		if ( this.originalEvent ) {
```

Comment:
Yeah, i have same feelings, i thought maybe you could think of the case :-). Even mutation and history events works here too, but i guess you never know.